app/uk/gov/hmrc/lisaapi/controllers/AccountController.scala

```diff
@@ -318,7 +350,7 @@ class AccountController extends LisaController with LisaConstants {
             path = getCloseEndpointUrl(lisaManager, accountId),
             auditData = closeLisaAccountRequest.toStringMap ++ Map(ZREF -> lisaManager,
               "accountId" -> accountId,
-              "reasonNotClosed" -> ErrorAccountAlreadyClosed.errorCode)
+              "reasonNotClosed" -> ErrorAccountAlreadyClosedOrVoid.errorCode)
```

Comment:
This goes against what's in the API spec, but I suspect it's correct.

Please check to see which is correct and update the spec or code accordingly.